graphrag/index/cli.py

```diff
@@ -318,3 +322,56 @@ def _enable_logging(root_dir: str, run_id: str, verbose: bool) -> None:
         datefmt="%H:%M:%S",
         level=logging.DEBUG if verbose else logging.INFO,
     )
+
+# Function to validate config file for model/deployment name typos
+def _validate_config_names(
+    root: str,
+    config: str | None,
+    reporter: ProgressReporter) -> None:
+        import os
+        from graphrag.config.enums import LLMType
+        from graphrag.index.llm import load_llm, load_llm_embeddings
+        from datashaper import NoopVerbCallbacks
+        from graphrag.llm.types import (
+            LLM,
+            CompletionInput,
+            CompletionLLM,
+            CompletionOutput,
+            LLMInput,
+            LLMOutput,
+        )
+        
+        # Fetch raw list of params from config file
+        parameters = _read_config_parameters(root, config, reporter)
+        
+        # Validate Chat LLM configs
+        test_llm = load_llm(
```

Comment:
Can you renamed this variable to `llm` to align with other parts of the codebase? Thanks